packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -28,11 +32,47 @@ interface CardHeaderWithCustomSubheadingProps extends CardHeaderRequiredProps {
   subheadingText: string;
```

Comment:
Should `headingTextFtlId` be replaced by `headingAndSubheadingFtlId` for clarity?